x/auth/types/account.go

```diff
@@ -30,7 +35,10 @@ func NewBaseAccount(address sdk.AccAddress, pubKey crypto.PubKey, accountNumber,
 		Sequence:      sequence,
 	}
 
-	acc.SetPubKey(pubKey)
+	err := acc.SetPubKey(pubKey)
+	if err != nil {
+		panic(err)
```

Comment:
auch. But I  guess we don't have other choice.

Reply:
indeed.